cglims/api.py

```diff
@@ -11,6 +11,49 @@ def connect(config):
     api = ClinicalLims(config['host'], config['username'], config['password'])
     return api
 
+class ClinicalSample:
+
+    def __init__(self, sample):
+        """ Wrapper around the genologics Sample class
+
+        Args:
+            sample (genologics.Sample): the sample instance to extend
+        """
+        self.lims = sample
+        self._pipeline = None
+        self._apptag = None
```

Comment:
The idea was that if we add more properties that do not rely on the apptag, then we don't need to have the apptag loaded on init.

What do you think? overkill?